test/test_torch.py

```diff
@@ -19024,6 +19024,46 @@ def make_index_negative(sequence, idx):
         np_fn = partial(np.moveaxis, source=(), destination=())
         self.compare_with_numpy(torch_fn, np_fn, x, device=None, dtype=None)
 
+    @dtypes(torch.int64, torch.float, torch.complex128)
+    def test_swapdim_invalid(self, device, dtype):
+        shape = self._rand_shape(4, min_size=5, max_size=10)
+        x = self._generate_input(shape, dtype, device, False)
+
+        # Invalid `source` and `destination` dimension
+        with self.assertRaisesRegex(IndexError, "Dimension out of range"):
+            torch.swapdim(x, 5, 0)
+
+        with self.assertRaisesRegex(IndexError, "Dimension out of range"):
+            torch.swapdim(x, 0, 5)
+
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
```

Comment:
We now require NumPy to run the test suite and these skips are no longer needed.